src/parsers/manifestjson.js

```diff
@@ -274,3 +280,7 @@ export default class ManifestJSONParser extends JSONParser {
     };
   }
 }
+
+export function getParsedJSON() {
+  return parsedJSONPermissions;
+}
```

Comment:
Instead of introducing a new global I think it's easier to just have the `JavaScriptScanner` parse the manifest again, set the permissions to `this.options.addonMetadata` and use that inside your rule. See `src/rules/javascript/webextension-api.js` as an example.